### PR TITLE
PHP 8.3 support added

### PR DESCRIPTION
**Pull request by glo71317**
|    Q          |   A
|-------------- | ------
| Documentation | yes/no
| Bugfix        | yes/no
| BC Break      | yes/no
| New Feature   | yes/no
| RFC           | yes/no
| QA            | yes/no

### CODE REVIEW

**Comment by glo71317** [comment]
@Ocramius @gsteel Can you review this PR if changes looks fine to you then please proceed for merging?

File: phpcs.xml.dist [hunk 1]
```diff
@@ -1,8 +1,307 @@
...
+    </rule>
+    <rule ref="PSR12.ControlStructures.ControlStructureSpacing.FirstExpressionLine">
+        <exclude-pattern>src/Token/AuthorizedRequest.php</exclude-pattern>
+    </rule>
+    <rule ref="SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse">
+        <exclude-pattern>src/Token/TokenInterface.php</exclude-pattern>
+    </rule>
+    <rule ref="SlevomatCodingStandard.TypeHints.ReturnTypeHint.MissingAnyTypeHint">
+        <exclude-pattern>src/Token/AbstractToken.php</exclude-pattern>
+    </rule>
```
Comment (Ocramius) [inline phpcs.xml.dist]: Similar question to https://github.com/laminas/laminas-file/pull/15#pullrequestreview-1729323169

/cc @laminas/technical-steering-committee

Comment (glo71317) [reply phpcs.xml.dist]: @Ocramius @gsteel 
I have addressed in this PR similar queries  https://github.com/laminas/laminas-file/pull/15#pullrequestreview-1729323169, Now can you please guide now how to proceed on this?

**Review by Ocramius — APPROVED** [review]
Even with a red CI job for CS checks, I'd say this is good to go.

Will squash-merge, just as a warning to involved contributors.

Thanks @glo71317!

**Pull request merged**